Python/pythonrun.c

```diff
@@ -1199,6 +1199,36 @@ print_exception_notes(struct exception_print_context *ctx, PyObject *notes)
     return -1;
 }
 
+static int
+get_exception_notes(struct exception_print_context *ctx, PyObject *value, PyObject **notes) {
+    PyObject *type;
+    PyObject *errvalue;
+    PyObject *tback;
+    PyObject *note = NULL;
+
+    if (_PyObject_LookupAttr(value, &_Py_ID(__notes__), notes) < 0) {
+        PyErr_Fetch(&type, &errvalue, &tback);
```

Comment:
PyErr_Fetch is deprecated. Let's use the replacement: 
https://docs.python.org/3/c-api/exceptions.html#c.PyErr_GetRaisedException

Reply:
Ah no, this is for 3.11. Ignore.